_posts/newsletters/2021-05-22-newsletter.md

```diff
@@ -0,0 +1,115 @@
+
+**Website**: The website working group is working on a new design for the us-rse.org website. The group is meeting bi-weekly and a beta version of a new website is in development. The new site promises to have better functionality and usability while still retaining the international RSE community feel. If you’d like to contribute please join the[ #website](https://usrse.slack.com/messages/website) channel in the US-RSE Slack to join in the conversation.
+
+<a name="workshop"></a>
+# **2021 US-RSE Virtual Workshop**
+
+Following the success of the 2020 US-RSE Virtual Workshop, the event this year is also virtual due to the ongoing pandemic. The [2021 US-RSE Virtual Workshop](https://us-rse.org/virtual-workshop-2021/) will be held on two non-consecutive half days: May 24 and 27, 2021, 1-4pm ET/12-3pm CT/10am-1pm PT.
+
+The theme of the workshop is "A Path forward for Research Software Engineers." The workshop will feature talks, community events, and working group sessions. It will be held via Zoom and some parts, such as the talks, will be recorded and available after the workshop via the [US-RSE YouTube channel](https://www.youtube.com/channel/UC7IQsWv809OQYJ-sJKuQZrw). The workshop is free but [registration is required, here](https://docs.google.com/forms/d/e/1FAIpQLSetiGdTiHSTfil4-LL_R3eZEyNIIfGtSJqjTsViBPcldsJtJw/viewform).
+
+***You can still submit breakout sessions by Monday before the workshop!*** See more details [here](https://us-rse.org/virtual-workshop-2021/participate/).
+
+<a name="501c3"></a>
+# **US-RSE Open Collective 501(c)(3) created**
+
+US-RSE has formalized its membership in the Open Collective Foundation (OCF) fiscal host entity. OCF provides us an open platform for transparently managing the US-RSE long-term finances. Next steps will include a blog post that describes the sorts of things this could facilitate, and a breakout discussion at the 2021 US-RSE Virtual Workshop on possible fiscal priorities for the US-RSE.
+
+<a name="communitycall"></a>
+# **US-RSE Community Calls**
+
+We will have our next community call on June 10, 2021 at 12-1pm ET. Have a look nearer to the date in the slack channel #communitycalls to vote for a topic! You can find the videos from past community calls in our YouTube channel. Here is the one from [April](https://www.youtube.com/watch?v=9LwPO2bdHRE). We did not have a call in May since we will have the Second US-RSE Virtual Workshop on Monday and Thursday (May 24 and May 27). See above for more information.
```

Comment:
```suggestion
We will have our next community call on June 10, 2021 at 12-1pm ET. Have a look nearer to the date in the slack channel [#communitycalls](https://usrse.slack.com/archives/CLBDQMJH5) to vote for a topic! You can find the videos from past community calls in our YouTube channel. Here is the one from [April](https://www.youtube.com/watch?v=9LwPO2bdHRE). We did not have a call in May since we will have the Second US-RSE Virtual Workshop on Monday and Thursday (May 24 and May 27). See above for more information.
```